src/routes/(admin)/dashboard/page.svelte.spec.ts

```
@@ -0,0 +1,78 @@
+					json: () =>
+						Promise.resolve({
+							total_test_submitted: 8,
+							total_test_not_submitted: 2,
+							not_submitted_active: 1,
+							not_submitted_inactive: 1
+						})
+				});
+			}
+
+			return Promise.reject(new Error('Unknown API'));
+		}) as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)